web/beacon-app/src/features/topics/components/TopicQuery.tsx

```diff
@@ -29,8 +33,24 @@ const TopicQuery = ({ name }: TopicNameProps) => {
           <div className="flex space-x-1">
             <p>
               <Trans>
-                Coming soon! Query the topic for insights with EnSQL e.g. the latest event or last
-                10 events.
+                Query the topic for insights with{' '}
+                <Link
+                  to={EXTRENAL_LINKS.ENSQL}
+                  className="font-semibold text-[#1D65A6] underline"
```

Comment:
A minor change from the Figma design. I think that making the links semi-bold will be consistent with how links are displayed elsewhere.